app/templates/introduction.html

```diff
@@ -25,14 +25,14 @@
             <div class="grid__col col-7@m">
 
                 <h2 class="saturn">You are completing this for
-                    <span>{{ meta.respondent.name}}</span>
-                    {% if meta.respondent.trading_as %}
-                        <span class="trading_as">(trading as&nbsp;<span class="trad_as">{{meta.respondent.trading_as }}</span>)</span>
+                    <span>{{ metadata.name}}</span>
```

Comment:
I think this should be `{{ metadata.ru_name }}` ?

Reply:
Good catch.